packages/bundler/src/parseScannerResult.ts

```diff
@@ -200,11 +200,25 @@ export function parseScannerResult (userOp: UserOperation, tracerResults: Bundle
     paymaster: validationResult.paymasterInfo
   }
 
+  // method-signature for entity calls.
+  const topLevelMethodSigs: {[key: string]: string} = {
+    factory: '0x570e1a36', // createSender
```

Comment:
Also here, I think it is worth it to hash here, with `ethers.id('transferFrom(address,address,uint256)'` or something.